docs/site/Controllers.md

```diff
@@ -122,104 +148,125 @@ The `@api` decorator allows you to annotate your Controller with a specification
           '200': {
             description: 'greeting text',
             schema: {type: 'string'},
-          }
-        }
-      }
-    }
-  }
+          },
+        },
+      },
+    },
+  },
 })
 class MyController {
-  greet(name) {
+  greet(name: string) {
+    return `hello ${name}`;
   }
 }
 app.controller(MyController);
 ```
 
 ## Writing Controller methods
 
-Below is an example Controller that uses several built in helpers (decorators). These helpers give LoopBack hints about the Controller methods.
+Below is an example Controller that uses several built in helpers (decorators).
+These helpers give LoopBack hints about the Controller methods.
 
 ```js
-import 'HelloRepostory' from 'path.to.repository';
-import 'HelloMessage' from 'path.to.type';
+import {HelloRepository} from '../repositories';
+import {HelloMessage} from '../models';
+import {get, param} from '@loopback/rest';
+import {repository} from '@loopback/repository';
 
-class HelloController {
-  constructor() {
-    this.repository = new HelloRepository(); // our repository
-  }
+export class HelloController {
+  constructor(
+    @repository(HelloRepository.name) protected repository: HelloRepository,
+  ) {}
+
+  // returns a list of our objects
   @get('/messages')
-  @param.query.number('limit')
-  async list(limit = 10): Promise<HelloMessage[]> { // returns a list of our objects
+  async list(@param.query.number('limit') limit = 10): Promise<HelloMessage[]> {
     if (limit > 100) limit = 100; // your logic
     return await this.repository.find({limit}); // a CRUD method from our repository
   }
 }
 ```
 
-- `HelloRepository` extends from `Repository`, which is LoopBack's database abstraction. See [Repositories](./Repositories.md) for more.
-- `HelloMessage` is the arbitrary object that `list` returns a list of.
-- `@get('/messages')` creates the `Route` for the Operation using `app.route()`.
-- `@param.query.number` adds a `number` param with a source of `query`.
+* `HelloRepository` extends from `Repository`, which is LoopBack's database
+  abstraction. See [Repositories](./Repositories.md) for more.
+* `HelloMessage` is the arbitrary object that `list` returns a list of.
+* `@get('/messages')` automatically creates the
+  [Paths Item Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#path-item-object)
```

Comment:
What does this apply to in the above example?

Reply:
The message I wanted to get across here was that `@get` is used to create a route spec. Was it confusing?

Reply:
I think the formatting was just weird and tripped me up.